app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,19 @@
+class RecipeFoodsController < ApplicationController
+  def new
```

Comment:
- [ ] Your code does not solve `N+1` query problem. I suggest you use `includes` method for eager loading. An example of this is doing something similar to:
```ruby
@posts = @user.posts.includes(:comments)
```
 instead of just
 ```ruby
 @posts = @user.posts
```